monorepo/scripts/syncPackages.js

```diff
@@ -0,0 +1,43 @@
+const { logInfo, readFile, writeFile, readFiles, path } = require('./utils');
+const prettier = require('prettier');
+
+const mainPackage = './package.json';
+const packages = [
+    /*'./package-lock.json'*/
+];
+
+const main = async function() {
+    let mainVersion = JSON.parse(await readFile(mainPackage)).version;
```

Comment:
man, what's all this? ;-) this reminds me of webpack ;-)

Reply:
removed, was me getting carried away :-)